common/core/overlaylayer.cpp

```diff
@@ -571,7 +571,7 @@ void OverlayLayer::ValidatePreviousFrameState(OverlayLayer* rhs,
 
   if (!layer->HasVisibleRegionChanged() && !content_changed &&
       surface_damage_.empty() && !layer->HasLayerContentChanged() &&
-      !(state_ & kNeedsReValidation)) {
+      !(state_ & kNeedsReValidation) && !layer->GetUseForMosaic()) {
```

Comment:
Still wondering why this issue is related to mosaic display?

Reply:
The property "total_display_" is only set in mosaic mode from ‘common/core/mosaicdisplay.cpp’. with the original 'if (total_displays_ == 1) {", the layer status on first display will not be set correctly, and the surface-damage will not be transferred.